docs/product-manuals/zeebe/deployment-guide/getting-started/deploy-a-process.md

```diff
@@ -3,9 +3,11 @@ id: deploy-a-process
 title: "Deploy a process"
 ---
 
-_In this section, we're going to start up the Zeebe broker as well as Camunda Operate, a tool that gives you visibility into deployed processes and running process instances and contains tooling for fixing problems in those process instances._
+In this section, we'll start up the Zeebe broker and Camunda Operate—a tool that gives you visibility into deployed processes and running process instances, and contains tooling for fixing problems in those process instances.
```

Comment:
:thought_balloon: With the new phrasing connection "and" it is less clear to me that Operate is the "tool that gives you visibility". Arguably, it maybe wasn't all that clear before.

So maybe something along the lines of: "In this section we'll start up the Zeebe broker - the underlying worklfow execution engine - and Operate - a tool that gives you..."

But then there might be too many hyphens. 

Also not sure why it's Zeebe broker but **Caumunda**  Operate since both are Camunda products. But that is more a marketing thing.

Reply:
Agreed on this -- your proposed text is much more clear. Added into latest commit.